### ISSUE TITLE
Implement mock Discord Social Credential workflow based on mock design

### ISSUE THREAD

**Issue by 2byrds** [roots-id/roots-react-native#21]
We should see the messages (narration, request, response, and action messages).
And the credential should look like:
<img width="1035" alt="image" src="https://user-images.githubusercontent.com/681493/202918334-f8ab3e59-f381-41a5-bd24-cca78d393c6a.png">

**Comment by HunnySajid** [comment]
@2byrds  are we going to scan any legitimate QRcode for discord specifically or we can use dummy for that as well?

**Comment by 2byrds** [comment]
no need for real data.... For this first iteration, lets just have a contact named 'Discord Social Issuer' (use a discord icon as the avatar) that automatically exists in your contact list (eventually we might make a way to add it).  in the chat of 'Discord Social Issuer' there is a message telling you that your 'Your Discord Social credential has been created for you'.  With three buttons 'Preview', 'Accept', 'Deny'.  Preview should show the credential details (but not add it to credentials), Accept should add the credential to our list and post a message saying 'Discord Social credential accepted', with a 'View' button.  Deny should respond with a message 'Discord social credential denied'.
We may add some additional narration, etc. after we demo it to the team.  But this is enough for this issue

**Comment by 2byrds** [comment]
![image](https://user-images.githubusercontent.com/681493/203321192-031301e9-266c-4fb7-98c7-83358df350a7.png)

**Comment by 2byrds** [comment]
@HunnySajid can you also try to clean up the warnings in the current build, as part of this work.  it seems there are circular dependencies related to naviagtion constants, etc. We want this repo to be very easy to approach and first time users have questioned if something is wrong, because of the current warnings.

**Comment by HunnySajid** [comment]
> @HunnySajid can you also try to clean up the warnings in the current build, as part of this work. it seems there are circular dependencies related to naviagtion constants, etc. We want this repo to be very easy to approach and first time users have questioned if something is wrong, because of the current warnings.

Sure, I will look into it

**Comment by HunnySajid** [comment]
@2byrds I have disabled the warning, for now, and will reduce the number of warnings in parallel to the upcoming tasks gradually.

**Comment by 2byrds** [comment]
> @2byrds I have disabled the warning, for now, and will reduce the number of warnings in parallel to the upcoming tasks gradually.

Excellent thank you!

**Comment by 2byrds** [comment]
This is good for now.... even though it doesn't show the workflow from my slides, lets do that in another issue once i can tell you what the workflow data will look like.
